glyphs-reader/src/propagate_anchors.rs

```diff
@@ -22,14 +22,24 @@ impl Font {
     pub fn propagate_all_anchors(&mut self) {
         propagate_all_anchors_impl(&mut self.glyphs);
     }
+
+    /// returns a list of all glyphs, sorted by component depth.
```

Comment:
all glyphs or only composites? - the commen tsays all, the fn name suggests only composites